data/projects/create_jenkins_plugin.md

```diff
@@ -6,6 +6,20 @@ If you find this fun you'll be the one knowing this component best and will
 get the chance to maintain it on your own if you want - always with the
 support of the community guarding your back of course.
 
+#### Motivation
+
+Jenkins is the most popular ci-cd server, and there are many corporations 
+who already use it. Integrating coala into Jenkins can attract business 
+clients better who could end up sponsoring/helping out coala.
+
+#### Final Goal
+
+The aim of the project is to make a plugin for Jenkins which can do 
+static analysis on code and make these visible to the user. coala 
+provides a framework for static analysis along with modularized 
+analyzers called bears. In the project, coala will be extended to 
```

Comment:
hah: `coala provides a framework for static analysis along with modularized analyzers called bears.` why are you dumping a random coala description into the goal? :rofl: